frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -24,39 +22,46 @@ import {
 import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
 import { eventWithTime } from '@rrweb/types'
 import { Dayjs, dayjs } from 'lib/dayjs'
-import {
-    getPlayerPositionFromEpochTime,
-    getPlayerTimeFromPlayerPosition,
-    guessPlayerPositionFromEpochTimeWithoutWindowId,
-} from './playerUtils'
 import type { sessionRecordingDataLogicType } from './sessionRecordingDataLogicType'
 import { teamLogic } from 'scenes/teamLogic'
 import { userLogic } from 'scenes/userLogic'
+import { chainToElements } from 'lib/utils/elements-chain'
 
 const IS_TEST_MODE = process.env.NODE_ENV === 'test'
 const BUFFER_MS = 60000 // +- before and after start and end of a recording to query for.
 
 export const parseMetadataResponse = (recording: SessionRecordingType): SessionRecordingMeta => {
+    let startTimestamp: Dayjs | undefined
+    let endTimestamp: Dayjs | undefined
+
     const segments: RecordingSegment[] =
         recording.segments?.map((segment): RecordingSegment => {
-            const windowStartTime = +dayjs(recording.start_and_end_times_by_window_id?.[segment.window_id]?.start_time)
-            const startTimeEpochMs = +dayjs(segment?.start_time)
-            const endTimeEpochMs = +dayjs(segment?.end_time)
+            const windowStartTime = dayjs(recording.start_and_end_times_by_window_id?.[segment.window_id]?.start_time)
+            const segmentStartTime = dayjs(segment.start_time)
+            const segmentEndTime = dayjs(segment.end_time)
             const startPlayerPosition: PlayerPosition = {
                 windowId: segment.window_id,
-                time: startTimeEpochMs - windowStartTime,
+                time: +segmentStartTime - +windowStartTime,
```

Comment:
Definitely nitpicking but... dayjs has a `diff` method. `+x - +y` confuses my brain 

But the not-a-plus-in-front API is almost equally confusing 🤷 

<img width="302" alt="Screenshot 2023-04-26 at 12 44 40" src="https://user-images.githubusercontent.com/984817/234564977-1468ca67-0317-4c02-a32e-3c99a0fd7edc.png">

If I find a magic lamp I'll use one of my three wishes for `dayJS(dateString).millisecondsSinceEpoch()` and then we can change this

Reply:
Yeah fair enough the +'s are arguably not as clear.